### ISSUE TITLE
Wrong expansions when using Alacritty on Linux

### ISSUE THREAD

**Issue by YakBarber** [federico-terzi/espanso#787]
OS: Ubuntu 20.04
Version: snap package, 0.7.3

When matching in my terminal (`alacritty`, with both `zsh` and `bash`), once in a while the replace text output comes in the wrong order. Examples I've seen:

`:espanso` -> `Hi t!here`
`:espanso` -> `!eHi ther`
`:espanso` -> `Hi thereespanso detectHi there there` (Note, `espanso detect` was the most recent command, so there was a scroll up in my shell history)

I am not able to reproduce the issue outside of the terminal (eg, in this text box in my browser it works right 100%).

**Comment by usernotnull** [comment]
@federico-terzi I moved to Linux last month, migrating from windows.
I never had issues with espanso there.
However, filling out the passport details on the Brave browser when I was buying the plane tickets, espanso filled in the numbers in the wrong order.
I was lucky enough to catch the mistake at the last stage before submitting :))

I tried to fill them here now on this github textarea and had no issues, and tried on VSCode and no issues. I realize this issue was in 2021, but any insight on how to resolve this to make sure the paste is 100% accurate?
Should I change the `backend` to Clipboard on linux? I'm on Pop_os, so X11. I do remember seeing also many other options specifically for x11 in the docs.
